packages/ds-component/src/index.ts

```diff
@@ -384,6 +384,34 @@ export const objectTransform: ObjectTransform = (message: Message) => ({
   interactions: transformDSInteraction(message),
 });
 
+/**
+ * Check if the transform is likely the identity function
+ * This is not a supported implementation path
+ * Avoid this if at all possible - please use either objectTransform or tableTransform
+ */
+const isIdentityFunction = (transform): boolean => {
+  let isIdentity: boolean = false;
+  if (transform('identity') === 'identity') {
+    isIdentity = true;
+    console.warn(
+      'This is an unsupported data format. Please use one of the supported transforms. Learn more: https://developers.google.com/datastudio/visualization/library-reference'
```

Comment:
Can we include either links or the names of the supported transforms. It'd be nice to include it here and maybe save some developer time not needing to jump between docs and code.

Reply:
I'll leave names of the transforms.